src/main/java/org/wso2/carbon/apimgt/securityenforcer/executors/PingAIExecutor.java

```diff
@@ -109,9 +109,12 @@ public boolean execute(RequestContext context, String currentState, String targe
                     return false;
                 }
                 GenericArtifact apiArtifact = artifactManager.getGenericArtifact(artifactId);
-                String apiName = apiArtifact.getAttribute(AISecurityHandlerConstants.ARTIFACT_ATTRIBUTE_API_NAME)
-                        + AISecurityHandlerConstants.API_NAME_VERSION_CONNECTOR + apiArtifact
-                        .getAttribute(AISecurityHandlerConstants.ARTIFACT_ATTRIBUTE_API_VERSION);
+                String apiName = apiArtifact.getAttribute(AISecurityHandlerConstants.ARTIFACT_ATTRIBUTE_API_NAME);
+                String apiVersion = apiArtifact.getAttribute(AISecurityHandlerConstants.ARTIFACT_ATTRIBUTE_API_VERSION);
+
+                //replace "." from version with "_" as from v4, ase does not support "." with version.
+                String modelName = apiName + AISecurityHandlerConstants.API_NAME_VERSION_CONNECTOR + apiVersion.replace(".","_");
```

Comment:
formatting issue ( space needed after the comma)
Also please check if this exceeds 120 line break

Reply:
Fixed with https://github.com/wso2-extensions/apim-handler-pingai/pull/22